interface/billing/payment_master.inc.php

```diff
@@ -341,7 +341,7 @@ function generate_list_payment_category($tag_name, $list_id, $currvalue, $title,
                         </div>
                         <div class="forms col-3">
                             <label class="control-label" for="TdUnappliedAmount"><?php echo xlt('Undistributed'); ?>:</label>
-                            <div  id="TdUnappliedAmount" class="form-control bg-danger"><?php echo ($UndistributedAmount*1==0)? attr("0.00") : attr(number_format($UndistributedAmount, 2, '.', ','));?></div>
+                            <div  id="TdUnappliedAmount" class="form-control bg-danger text-light"><?php echo ($UndistributedAmount*1==0)? attr("0.00") : attr(number_format($UndistributedAmount, 2, '.', ','));?></div>
```

Comment:
This div has one unnecessary space.

Reply:
@stu01509 Where I think  its same, only I have added a class.